pkg/agent/runner.go

```diff
@@ -830,7 +830,12 @@ func (r *Runner) DoSchedulerRequest(
 		return nil, fmt.Errorf("Bad JSON response: %w", err)
 	}
 
-	logger.Debug("Received response from scheduler", zap.Any("response", respData))
+	if respData.Permit.HasFieldLessThan(resources) {
+		r.global.metrics.schedulerSchedulingDeniedResponses.Inc()
+		logger.Warn("Received response from scheduler, permitted resources are insufficient", zap.Any("requested", resources), zap.Any("response", respData))
```

Comment:
WDYT about keeping the same log message `"Received response from scheduler", zap.Any("response", respData),  zap.Any("requested", resources)`, but changing the log level? 

This way it can be easily filtered in loki by any predicate: `response < requested` or  `response > 0` or `response == 0`. 

CC @sharnoff for thoughts

Reply:
In general, it sounds good, but does it mean we will have a warning-level message for each response, unconditionally, if it fails or succeeds?

Reply:
Not necessarily, we can have the same log message, but printed with a different log level.